metadata-ingestion/src/datahub/ingestion/source/aws/glue.py

```diff
@@ -48,6 +48,7 @@
 
 class GlueSourceConfig(AwsSourceConfig):
 
+    extract_owners: Optional[bool] = True
```

Comment:
Any strong reason to leave this true? 

In my opinion, it makes sense to play it safe. Otherwise we'll continue to get questions and reports that owners are being overwritten